electrumx/server/mempool.py

```diff
@@ -426,6 +426,26 @@ def _deserialize_txs(self, hashes, raw_txs):    # This function is pure
                                               0, tx_size, tx.type)
         return txs
 
+    async def balance_delta(self, hashX):
```

Comment:
I don't know python that much but this `async` seems to be not needed

Reply:
its function override, I rather leave it as it is in the parent class